### PR TITLE
Enable transfers

### PR DESCRIPTION
**Pull request by gavofyork**
Also quash any conviction from Referendum Zero; Sudo was always
going to have been removed so lock-voting doesn't make sense in
this case.

### CODE REVIEW

File: runtime/polkadot/src/lib.rs [hunk 1]
```diff
@@ -118,17 +118,15 @@ impl Filter<Call> for BaseFilter {
 			Call::Parachains(parachains::Call::set_heads(..)) => true,
 
 			// Parachains stuff
-			Call::Parachains(_) | Call::Attestations(_) | Call::Slots(_) | Call::Registrar(_) |
+			Call::Parachains(_) | Call::Attestations(_) | Call::Slots(_) | Call::Registrar(_) =>
 			// Balances and Vesting's transfer (which can be used to transfer)
```
Comment (sorpaas) [inline runtime/polkadot/src/lib.rs:122]: This line of comment looks not needed / not applicable any more.

File: runtime/polkadot/src/lib.rs [hunk 2]
```diff
@@ -487,7 +477,7 @@ parameter_types! {
 	pub const CandidacyBond: Balance = 100 * DOLLARS;
 	pub const VotingBond: Balance = 5 * DOLLARS;
 	/// Daily council elections initially, later weekly and monthly.
```
Comment (shawntabrizi) [inline runtime/polkadot/src/lib.rs:479]:
```suggestion
	/// Weekly council elections, later monthly.
```

File: runtime/polkadot/src/lib.rs [hunk 3]
```diff
@@ -36,18 +36,12 @@ use primitives::v0::{
...
-		BlakeTwo256, Block as BlockT, OpaqueKeys, ConvertInto, IdentityLookup,
-		Extrinsic as ExtrinsicT, SaturatedConversion, Verify,
-	},
-};
+use sp_runtime::{create_runtime_str, generic, impl_opaque_keys, ModuleId, ApplyExtrinsicResult, KeyTypeId, Percent, Permill, Perbill, transaction_validity::{
+	TransactionValidity, TransactionSource, TransactionPriority,
+}, curve::PiecewiseLinear, traits::{
+	BlakeTwo256, Block as BlockT, OpaqueKeys, ConvertInto, IdentityLookup,
+	Extrinsic as ExtrinsicT, SaturatedConversion, Verify,
+}};
```
Comment (gnunicorn) [inline runtime/polkadot/src/lib.rs:44]: Improved readability.

```suggestion
use sp_runtime::{
	create_runtime_str, generic, impl_opaque_keys, ModuleId, ApplyExtrinsicResult, KeyTypeId, Percent, Permill, Perbill,
	curve::PiecewiseLinear,
	transaction_validity::{
		TransactionValidity, TransactionSource, TransactionPriority,
	}, 
	traits::{
		BlakeTwo256, OpaqueKeys, ConvertInto, IdentityLookup, SaturatedConversion, Verify,
		Extrinsic as ExtrinsicT, Block as BlockT,
	},
};
```

Comment (gavofyork) [reply runtime/polkadot/src/lib.rs:44]: Happy with the changes but this wasn't actively part of the PR, so maybe resubmit as a pure whitespace PR?

**Pull request merged**